mp/src/game/shared/momentum/mom_shareddefs.h

```diff
@@ -185,6 +185,17 @@ const char* const g_szTimesStatusStrings[] = {
     "#MOM_API_UnauthFriendsList", // STATUS_UNAUTHORIZED_FRIENDS_LIST
 };
 
+enum LocalTimesStatus_t
```

Comment:
Also, this enum should probably be called something more generic like `AsyncReplayStatus_t` and start with `ASYNC_REPLAYS_` .